dissect/volume/vss.py

```diff
@@ -361,10 +352,7 @@ def read_block(self, block, active_store=None):
         if not buf:
             print("booboo")
             print("Descriptor", descriptor)
-            import ipdb
-
-            ipdb.set_trace()
-            raise Exception()
+            raise ValueError("Error reading block")
 
         if descriptor and active_store is self and (descriptor.is_overlay or descriptor.overlay):
             # print("overlaying data")
```

Comment:
```suggestion
```

Reply:
Possibly, also the commented code below on line 371

Reply:
I left the rest as it was since this file is still a WIP 😉 those are hints for whoever continues on it.